include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -3616,48 +3619,65 @@ struct OrtApi {
 
   /** \brief Register custom ops from a shared library.
-  *
-  * Loads a shared library (.dll on windows, .so on linux, etc) named 'library_name' and looks for this entry point:
-  *		OrtStatus* RegisterCustomOps(OrtSessionOptions * options, const OrtApiBase* api);
-  * It then passes in the provided session options to this function along with the api base.
-  *
-  * The handle to the loaded library is automatically released by ORT when the last OrtSession that references the
-  * library handle is released. If no OrtSession is created, then the library handle is released when the provided
-  * OrtSessionOptions is released.
-  *
-  * \param[in] options The session options.
-  * \param[in] library_name The name of the shared library to load and register. Refer to OS-specific dynamic library
-  *                         loading utilities (e.g., LoadLibraryEx on Windows or dlopen on Linux/MacOS) for information
-  *                         on the format of library names and search paths.
-  *
-  * \snippet{doc} snippets.dox OrtStatus Return Value
-  */
+   *
+   * Loads a shared library (.dll on windows, .so on linux, etc) named 'library_name' and looks for this entry point:
+   *		OrtStatus* RegisterCustomOps(OrtSessionOptions * options, const OrtApiBase* api);
+   * It then passes in the provided session options to this function along with the api base.
+   *
+   * The handle to the loaded library is automatically released by ORT when the last OrtSession that references the
+   * library handle is released. If no OrtSession is created, then the library handle is released when the provided
+   * OrtSessionOptions is released.
+   *
+   * \param[in] options The session options.
+   * \param[in] library_name The name of the shared library to load and register. Refer to OS-specific dynamic library
+   *                         loading utilities (e.g., LoadLibraryEx on Windows or dlopen on Linux/MacOS) for information
+   *                         on the format of library names and search paths.
+   *
+   * \snippet{doc} snippets.dox OrtStatus Return Value
+   */
   ORT_API2_STATUS(RegisterCustomOpsLibrary_V2, _Inout_ OrtSessionOptions* options, _In_ const ORTCHAR_T* library_name);
 
+  /** \brief Register custom ops by calling a RegisterCustomOpsFn function.
+   *
+   * Searches for registration_func_name and if found calls it. The library containing the function must either be
+   * linked against or previously loaded by the executable.
+   * If you require ONNX Runtime to load the library, use RegisterCustomOpsLibrary_V2.
+   *
+   * The registration function must have the signature of RegisterCustomOpsFn:
+   *    OrtStatus* (*fn)(OrtSessionOptions* options, const OrtApiBase* api);
+   *
+   * \param[in] options OrtSessionOptions to pass to registration function.
+   * \param[in] registration_func_name Name of registration function to use.
+   *
+   * \snippet{doc} snippets.dox OrtStatus Return Value
```

Comment:
nit: add `\since Version 1.14`

this should probably also be added to RegisterCustomOpsLibrary_V2